sdk/all/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -47,13 +47,20 @@ public static OpenTelemetrySdk get() {
 
   /** Returns the global {@link TracerSdkManagement}. */
   public static TracerSdkManagement getGlobalTracerManagement() {
-    TracerProvider tracerProvider = OpenTelemetry.get().getTracerProvider();
+    OpenTelemetry openTelemetry = OpenTelemetry.get();
+    if (openTelemetry instanceof OpenTelemetrySdk) {
+      return ((OpenTelemetrySdk) openTelemetry).tracerSdkManagement;
+    }
+    TracerProvider tracerProvider = openTelemetry.getTracerProvider();
+    if (tracerProvider instanceof TracerSdkManagement) {
+      return (TracerSdkManagement) tracerProvider;
+    }
     if (!(tracerProvider instanceof ObfuscatedTracerProvider)) {
       throw new IllegalStateException(
```

Comment:
Should we just return a no-op `TracerSdkManagement` instead of throwing an exception, perhaps with a debug log?

Reply:
I'm not sure. Would that possibly hide problems and make it more confusing for people to debug if/when things weren't working?

Reply:
Setting a tracerprovider that isn't an SDK one seems pretty explicit so Just logging and retuning a no-op seems like a reasonable fallback for this incorrect usage I guess (incorrect usage being it's not allowed to call `getTracerManagement` if the registered provider is not an SDK tracer, either because it's default or custom).